src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/PerformanceFormHandler.php

```diff
@@ -55,8 +55,7 @@ public function __construct(
         FormFactoryInterface $formFactory,
         FormDataProviderInterface $formDataProvider,
         CombinationFeature $combinationFeature
-    )
-    {
+    ) {
```

Comment:
something is wrong here, maybe an error with a rebase or a merge between branches, but AbstractFormHandler class doesnt exists.

Reply:
nevermind this class is deprecated.